src/network/networkd-manager.c

```diff
@@ -1454,8 +1454,14 @@ int manager_rtnl_enumerate_rules(Manager *m) {
                 return r;
 
         r = sd_netlink_call(m->rtnl, req, 0, &reply);
-        if (r < 0)
+        if (r < 0) {
+                if (r == -EOPNOTSUPP) {
+                        log_debug("FIB Rules is not supported by the kernel. Ignoring.");
```

Comment:
"FIB rules are not supported by the kernel, ignoring."

Reply:
rupdated